api/sql/schema/350-rule.conditionProperty.sql

```diff
@@ -5,7 +5,7 @@ CREATE TABLE [rule].[conditionProperty] (
     [value] NVARCHAR(200) NOT NULL,
     CONSTRAINT pkRuleConditionProperty PRIMARY KEY CLUSTERED (conditionId, factor, [name], [value]),
     CONSTRAINT fkRuleConditionProperty_conditionId FOREIGN KEY (conditionId) REFERENCES [rule].[condition](conditionId),
-    CONSTRAINT ccRuleConditionProperty_factor1 CHECK (factor IN ('so', 'do', 'co', 'ss', 'ds', 'cs', 'oc', 'sc', 'dc', 'sk', 'st', 'dk', 'dt'))
+    CONSTRAINT ccRuleConditionProperty_factor1 CHECK (factor IN ('so', 'do', 'co', 'ss', 'ds', 'cs', 'oc', 'sc', 'dc', 'sk', 'st', 'dk', 'dt', 'sp', 'dp'))
```

Comment:
I think this should be done in conditionItem?

Reply:
It is fixed with the last commit